### ISSUE TITLE
DCC Icons should be consistently colored within light and dark mode

### ISSUE THREAD

**Issue by bmuramatsu** [digitalcredentials/learner-credential-wallet#492]
Icons are differently colored between the Home screen, Bottom Nav Share screen, Add screen, Settings screen in both light and dark modes.

I prefer the higher contrast ("brighter") icon colors. One example below.

**Example difference in icon colors**
<img src="https://github.com/digitalcredentials/learner-credential-wallet/assets/752326/4dff5e7f-74cd-435d-b3be-f143364faac3" width=30%><img src="https://github.com/digitalcredentials/learner-credential-wallet/assets/752326/b3951ad6-a7c1-4c61-9f95-e1aab34069dd" width=30%>
